src/main/java/jenkins/security/plugins/ldap/LDAPGroupMembershipStrategy.java

```diff
@@ -64,4 +68,15 @@ public void setAuthoritiesPopulator(LdapAuthoritiesPopulator authoritiesPopulato
      * @return the {@link GrantedAuthority}s that the specified user belongs to.
      */
     public abstract GrantedAuthority[] getGrantedAuthorities(LdapUserDetails ldapUser);
+
+    /**
+     * Returns a {@link Set} of all members in the specified group.
+     *
+     * @param groupDn the DN of the group whose members will be returned.
+     * @param conf the {@link LDAPConfiguration} that controls some search variables.
+     * @return a set of all members in the specified group.
+     * @throws UsernameNotFoundException if a group with the specified DN does not exist.
+     * @throws DataAccessException if there is an error performing the search.
+     */
+    public abstract @CheckForNull Set<String> getGroupMembers(String groupDn, LDAPConfiguration conf) throws DataAccessException;
```

Comment:
Do I need to make this non-abstract to avoid breaking source compatibility with any third-party implementations of `LDAPGroupMembershipStrategy`?

Reply:
I think I should also add `@since 1.18`